### PR TITLE
Fixes #14105 - fixes backup/restore after moving to foreman-installer, BZ1316567

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Comment by ehelms** [comment]
Does this need to go in 3.0?

**Comment by cfouant** [comment]
@ehelms - yes

**Comment by ehelms** [comment]
Could you attach an issue to it since it is a code change that I'll need to pull into the release?

**Comment by cfouant** [comment]
@ehelms - updated, thanks!

File: katello/katello-backup [hunk 1]
```diff
@@ -24,37 +24,30 @@ else
...
-    '/etc/tomcat*',
-    '/root/ssl-build',
-    '/var/lib/foreman',
-    '/var/lib/katello',
-    '/var/lib/candlepin',
-    '/var/www/html/pub',
-    '/var/lib/puppet/foreman_cache_data',
-    '/var/lib/puppet/ssl'
-  ]
+CONFIGS=[
```
Comment (ehelms) [inline katello/katello-backup]: I think this should be indented still given its inside an else block

**Comment by cfouant** [comment]
@ehelms - added a BZ and fixed indentation. Thanks!

**Comment by ehelms** [comment]
This looks good to me, I had an after thought to get @mbacovsky to give this a quick once over to ensure nothing is missing with the structure changes. I'll give it another day for anything.

**Comment by mbacovsky** [comment]
It looks good to merge. I'm not sure about the `/var/lib/{katello,foreman}` dirs. They do not seem to be related to the installer.

**Comment by cfouant** [comment]
@mbacovsky @ehelms alright, also removed /var/lib/foreman (/var/lib/katello was already being removed).

**Comment by ehelms** [comment]
ACK - thanks @cfouant !

**Pull request merged**